src/renderer/components/uielements/poolStatus/PoolStatus.tsx

```diff
@@ -1,35 +1,35 @@
 import React from 'react'
 
-import { AssetAmount, formatAssetAmountCurrency } from '@thorchain/asgardex-util'
 import { Row } from 'antd'
 import BigNumber from 'bignumber.js'
 
 import Label from '../label'
 import Trend from '../trend'
-import { PoolStatusWrapper } from './PoolStatus.style'
+import * as Styled from './PoolStatus.style'
 
 type Props = {
-  amount: AssetAmount
   asset: string
   label: string
   target: string
-  trend: BigNumber
+  trend?: BigNumber
```

Comment:
It seems layout has been changed for BEPSwap at the meantime and we have to follow it for `PoolStatus`:

- Instead of `asset` / `target` it has just a currency label (depending on selected price)
- It seems no `trend` anymore (or is it still optional?) 

![Screenshot from 2020-09-21 17-38-04](https://user-images.githubusercontent.com/61792675/93788827-6701f500-fc31-11ea-8336-d734458dea9f.png)

Could you update `PoolStatus`, pls?

Reply:
sure.  trend is optional there. just forgot to add it. 
also here a question. as bepswap does not have it right now do wee need to add it at the thorchain ?

Reply:
We don't need to handle `trend` if BEPSwap does not display it anymore.